### ISSUE TITLE
Unable to retrieve the verification code when trying to use an e-mail to get your token

### ISSUE THREAD

**Issue by ryo-yamada** [kramcat/CharacterAI#116]
Character.ai doesn't send you a verification code anymore. Instead, it sends you a link you have to click to verify your email (which obviously renders the e-mail login method useless)![image](https://github.com/user-attachments/assets/b3e0d088-a0be-4fdf-a280-8b7257d8ba7e)

**Comment by Nytro69** [comment]
You're right, In order to get the token you have to go to the older version of charecter ai old.character.ai, then inspect the web page, go to local in system and you should see char_token, take the value parameter from it and thats your token

**Comment by ryo-yamada** [comment]
@Nytro69
Yeah, I know that, but it's just that I'm coding on mobile right now (which is an absolute pain) because I'm on vacation and my parents didn't let me take my laptop with me.

It's whatever though, I'll be home in about a week from now anyways.
